### DISCUSSION TITLE
Feature Request: button tile.

### DISCUSSION THREAD

**Discussion by tomlut** [home-assistant/frontend#14618] [Dashboard]
I'm going to attempt to replace my extensively themed frontend with nothing but core cards (and possibly the custom hui-element card). The main thing stopping me (apart from the poor graphing options) is the availability of a tile card with just an icon, like the button card. No state info, just the icon, an optional name and an action. For constructing remote controls with the grid card, like this:

![Screenshot 2022-12-08 at 13-32-15 Overview – Home Assistant](https://user-images.githubusercontent.com/10679300/206342054-5d086e77-65cb-485a-b3f2-052cf0e79677.png)

Would it be possible to add this sort of button tile?

**Comment by tomlut** [comment]
At the moment the closest I can get is:

![Screenshot 2022-12-08 at 09-55-20 Administration – Home Assistant](https://user-images.githubusercontent.com/10679300/206344941-0bb4622a-3453-4008-9b3e-9cbb49d5d04a.png)

**Comment by ChristophCaina** [comment]
why not just using the default button card?

with my custom theme (sharp edges)
![grafik](https://user-images.githubusercontent.com/26391061/206850932-69bda901-da74-4209-85e6-3ca9841e99ab.png)

with default HA theme (round edges)
![grafik](https://user-images.githubusercontent.com/26391061/206850982-826877a6-60f6-426c-bb32-4dfee4cfcec1.png)

Sorry, just have used the same entity for the demo, therefore - all icons are the same ... ^^

**Comment by tomlut** [comment]
Basically because it looks hideous compared to the tile card.

**Comment by BBE-FR** [comment]
For the time being the mushroom cards designed by @piitaya are capable to do this with almost the same design as the Tile card.

It would be a nice evolution of the tile card to be capable to hide also the name (maybe through YAML only) and not only the state.

Tile card with hidden name and state, with the capability to resize to 1x1 in the section, would be a perfect button, no need to a dedicated Tile button card...
![image](https://github.com/user-attachments/assets/c4c42021-de78-4234-aec8-15ffc67eb1d6)

**Comment by BBE-FR** [comment]
With the "simple" addition of the option to hide the name in the Tile card, we could create very simple "Tile buttons" !

This could be done the same way as the option to display/hide the state...

![image](https://github.com/user-attachments/assets/5b513d51-c638-411f-aa71-cf866adc23ed)

Mushroom from @piitaya are an easy way to display **Tile consistent** 1x1 buttons (here below a section header card):
![image](https://github.com/user-attachments/assets/17b81fbf-2c5d-4d8c-a2b6-5eaca381d947)

**Reply by Mariusthvdb** [reply]
well not the only. there is custom:button-card that is practically made for this. and much more configurable right out of the box

**Reply by BBE-FR** [reply]
Yes, but not as easy as mushroom or tile for the beginner.

Initial request being on Tile, mushroom (100% UI, same look, same developer) is closest in term of philosophy than button-card...

**Comment by TheFes** [comment]
Yes, I agree with @BBE-FR.
I'm trying to move away from custom cards as much as possible, and I now have tile cards with a space as the name (as completely removing the name will just display the name set to the entity).
An option to hide the name would also give Tom exactly what he wants, a tile card showing only an icon (as there are already options to hide the state)

**Comment by tomlut** [comment]
Likewise I am trying to minimise my dependence on 3rd party cards. 

There is a working group on Discord that are designing a remote card but despite my advice they seem to be fixated on adding a pre-defined list of button functions rather than the much more universal method of allowing the user to define any button icon and action 🙄

**Comment by Mariusthvdb** [comment]
> Likewise I am trying to minimise my dependence on 3rd party cards.

not sure if you are still  using `card-mod` ?

if yes, you could for the time being use:
```
card_mod:
  style:
    .: |
      ha-tile-info {
        display: none;
      }
```
it effectively takes out the space the name would have used too, so you are not left with an empty space on the <strike>button</strike> Tile

**Comment by piitaya** [comment]
Why not improving the design of the button card instead of adding option to tile card to act like a button card? 
I'm not in favor to add option to the tile card for multiple reasons : 
- the round shape around the icon is a button so we will be a button icon inside a button
- it will have the same feature as the button card, only the design will be different

**Reply by BBE-FR** [reply]
Improving the design of the default button card to be "Tile compatible" would be equivalent of course.

Looking to the section views, a 1x1 button compatible with the tile look is missing for the time being.

Concerning the button in a button, I have to admit that I always forget that the click on tile card and Tile icon have two different behavior (I was mushromised before trying to switch to tiles... thus this functionality, working well in mushroom, is more questionable on a 1x1 tile...)

**Reply by Mariusthvdb** [reply]
otoh, only adding the option to not show the name isnt that extravagant, and makes the card that much more versatile.

_especially_ leaving the rest of the design untouched, and beautiful as it is.

**Reply by BBE-FR** [reply]
That is true.

Being able to simply hide the name in the Tile card would make it both simple and versatile.
It will mimic the behavior of the badges or entities in the header.
![image](https://github.com/user-attachments/assets/378b1f3f-3aef-4937-9d09-d069ddfc7ce3)

**Reply by BBE-FR** [reply]
> otoh, only adding the option to not show the name isnt that extravagant, and makes the card that much more versatile.
> 
> _especially_ leaving the rest of the design untouched, and beautiful as it is.

This would also allow to display only the content without the name. The "button tile" would be a corner use.

**Comment by tomlut** [comment]
I'm pretty much happy with just the tile card as it stands now:

<img width="670" height="1561" alt="image" src="https://github.com/user-attachments/assets/11a610ac-1722-48ad-95b3-f2955e4e4c11" />

So this request is no longer needed.
